content/news/013.dd-ix-opening.de.md

```diff
@@ -0,0 +1,23 @@
+---
+published: 2024-11-28
+description: Vorträge, Snacks und viel gute Stimmung
+title: Die Feierliche Eröffnung des DD-IX
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 013_dd_ix_opening_press.webp
+---
+
+Am 25. November feierten wir unsere offizielle Eröffnung mit Freunden des DD-IX's und lokalen Internetakteuren. Die Veranstaltung fand 
+im SachsenEnergieCenter in der Nähe des Dresdner Hauptbahnhofs statt. 
+
+[Rack](013_dd_ix_opening_rack.webp)
+
+Wir möchten uns bei unseren Vortragenden: Prof. Matthias Wählisch, Prof. Thomas Popp, Thomas Eibeck, Prof. Thomas Horn, Vint Cerf, Christian Dieckhoff,
+Marc Korthaus, Markus Ziehe, und Mirjam Kühne für ihre herausragenden Vorträge bedanken. Wir möchten uns auch bei allen bedanken, die uns im Hintergrund unterstützt haben, wie Thomas Wild,
+Kerstin Fiering oder Tatjana Schiehandl von der SachsenGigaBit GmbH, die uns auch ihre Räumlichkeiten zur Verfügung gestellt haben. Wir möchten uns auch bei Florian Hibler, 
+René Fichtmüller, Michael Nitsche, Thomas Weible, Sebastian Neuner, Tim Kleefass, Kurt Jaeger und Moritz Frenzel für ihr großartiges Geschenk bedanken, was 
+uns bei der weiteren Entwicklung des DD-IX enorm helfen wird.
+
+![Rack](013_dd_ix_opening_team.webp)
+
+Es war eine großartige Erfahrung, euch alle nach Dresden einzuladen und mit euch diesen wichtig Schritt für den DD-IX zu feiern.
```

Comment:
wir wollen nicht ueber unsere Erfahrungen berichten ;). Besser: Wir haben uns sehr gefreut, euch alle in Dresden begrüßen zu können und diesen für den DD-IX wichtigen Schritt mit euch gemeinsam zu feiern. Community at its best!

Reply:
done